tests/test_flows.py

```diff
@@ -380,57 +380,44 @@ def test_channel_selection(self):
         self.check_log(self.repeater1, "agent_wlan0", "CHANNEL_PREFERENCE_QUERY_MESSAGE")
         self.check_log(self.repeater1, "agent_wlan2", "CHANNEL_PREFERENCE_QUERY_MESSAGE")
 
-        payload_empty="0x14"
+        # Send empty channel selection request
+        self.debug("Send empty channel selection request")
+        self.gateway_ucc.dev_send_1905(self.mac_repeater1,0x8006)
 
+        self.debug("Confirming empty channel selection request has been received on agent")
+        self.check_log(self.repeater1, "agent_wlan0", "CHANNEL_SELECTION_REQUEST_MESSAGE")
+        self.check_log(self.repeater1, "agent_wlan2", "CHANNEL_SELECTION_REQUEST_MESSAGE")
+
+        self.debug("Confirming OPERATING_CHANNEL_REPORT_MESSAGE message has been received on controller")
+        self.check_log(self.gateway, "controller", "OPERATING_CHANNEL_REPORT_MESSAGE")
+
+        # Send channel selection request with chnging tx_power_limit 
+        # Payload for transmit power (tp)
+        tp20dBm = "0x14"
+        tp21dBm = "0x15"
```

Comment:
I suggested before to write this as
```suggestion
        # Send channel selection request with changing tx_power_limit 
        # Payload for transmit power (tp)
        tp20dBm = 0x14
        tp21dBm = 0x15
```
You marked that as resolved with no feedback, so that sounds as if you accepted the proposal. But it's still here...